tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -2174,5 +2180,12 @@ def test_get_span_attribute_metrics(default_project: Project) -> None:
                     {"field": "span.data.bar", "key": "bar"},
                     {"field": "span.data.foo", "key": "foo"},
                 ],
-            }
+            },
+            {
+                "category": "span",
+                "condition": None,
```

Comment:
Would it make sense to add a test for something like empty-string or \s? A condition that doesn't do anything but is not None may use some different execution paths.

Reply:
[c81910b](https://github.com/getsentry/sentry/pull/73178/commits/c81910ba0b3e546cb109054f1df9f4adaee05d8d)